src/Client.php

```diff
@@ -326,12 +326,33 @@ public function generatePrivateChannelToken($client, $channel, $exp = 0, $info =
         return implode('.', $segments);
     }
 
-    private function send($method, $params = [])
+/*
+ * Function added for backward compatibility with PHP version < 5.5
+ */
+    
+    public function _json_last_error_msg() {
+      if (function_exists('json_last_error_msg')) {
+        return json_last_error_msg(json_last_error());
```

Comment:
Looking at https://www.php.net/manual/ru/function.json-last-error-msg.php json_last_error_msg definition is:
```php
json_last_error_msg ( void ) : string
```

Here you are calling it with argument `json_last_error()` - can you explain why?

Reply:
It's my mistake. Fixed.